templates/case-studies/index.html

```diff
@@ -1,6 +1,5 @@
-{% extends "case-studies/base_case-studies.html" %}
+{% extends "templates/base.html" %}
```

Comment:
I'm only seeing a header and footer on the demo, no hero content:

<img width="1204" alt="Screenshot 2024-08-16 at 8 57 02 AM" src="https://github.com/user-attachments/assets/d1da7cf6-5620-41a0-9396-a3f102e06fbf">

Reply:
The example case studies are on https://ubuntu-com-14166.demos.haus/case-studies/example after rebasing a recent merged PR, I've updated the PR description to reflect this. The index page should be an overview of case studies.